app/presenters/account_show_presenter.rb

```diff
@@ -34,10 +36,44 @@ def show_gpo_partial?
     user.gpo_verification_pending_profile?
   end
 
+  def show_ipp_partial?
+    user.pending_in_person_enrollment.present?
+  end
+
   def showing_any_partials?
+    show_password_reset_partial? ||
+      show_gpo_partial? || show_ipp_partial? || show_pii_partial?
+  end
+
+  def showing_alerts?
     show_service_provider_continue_partial? ||
-      show_password_reset_partial? ||
-      show_gpo_partial?
+      show_password_reset_partial?
+  end
+
+  def show_unverified?
+    show_password_reset_partial? &&
+      (!show_ipp_partial? || !show_gpo_partial?)
+  end
```

Comment:
I'm kinda confused what "unverified" is supposed to mean. As implemented here, it seems to be related to password reset. But as it appears in the Figma designs, it seems to be related to if the user started but hasn't yet completed identity verification, and the design itself looks different as well.

Current Branch|Figma Design
---|---
![image](https://github.com/18F/identity-idp/assets/1779930/6d2d8339-7bee-4c83-9b78-bacd7ba384aa)|![image](https://github.com/18F/identity-idp/assets/1779930/32147cde-6428-438e-9b74-c7bd9fc5c110)

Reply:
Related Slack discussion: https://gsa-tts.slack.com/archives/C05R6BLVAQG/p1719430728774569

>- Not changing the current production implementation for pending profile after password reset
>- Need to update pull request to implement behavior where “Unverified” badge is shown if user starts IdV but doesn’t complete it

Reply:
Updated in ac9737e087 to target incomplete IdV instead of password reset pending profile.